src/screens/OtherProfile/index.js

```diff
@@ -255,15 +263,15 @@ const OtherProfile = () => {
       if (dataMain.is_following)
         return (
           <GlobalButton onPress={() => handleSetUnFollow()}>
-            <View style={styles.buttonFollowing}>
-              <Text style={styles.textButtonFollowing}>Following</Text>
+            <View style={styles.buttonFollowing(isAnonimity)}>
+              <Text style={styles.textButtonFollowing(isAnonimity)}>Following</Text>
             </View>
           </GlobalButton>
         );
 
       return (
         <GlobalButton onPress={() => handleSetFollow()}>
-          <View style={styles.buttonFollow}>
+          <View style={styles.buttonFollow(isAnonimity)}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here are similar to those in line 208-223, where the color values are updated to use the new `COLORS` object from the `theme` module. Again, it's important to ensure that the new colors (`COLORS.anon_primary`, `COLORS.signed_primary`) are defined in the `COLORS` object and they match the intended design.



<!-- This is an auto-generated comment by OSS CodeRabbit -->